mapie/tests/test_quantile_regression.py

```diff
@@ -46,6 +46,15 @@
             random_state=random_state
             )
 
+
+def early_stopping_monitor(i, est, locals):
+    """Returns True on the 3rd iteration."""
+    if i == 2:
+        return True
+    else:
+        return False
```

Comment:
I also suggest moving it directly into the test to better understand where the function is being used.